src/patternfly/components/Alert/examples/alert-danger-example.hbs

```diff
@@ -3,27 +3,27 @@
   {{/alert-icon}}
   {{#> alert-body alert-body--attribute='role="alert"'}}
     {{#> alert-title}}
-      {{#> accessibility accessibility--type="sr-only"}}Danger: {{/accessibility}}
```

Comment:
should all of these still pull in the accessibility utility rather than just using the span?